pages/index.html

```diff
@@ -967,7 +967,7 @@ <h3 class="version-profile--name">Ethan Muller</h3>
       <section class="learn--dev">
         <h2 class="learn--header">Full-Stack Developer</h1>
         <p class="info-secondary learn--text">Take a dive into the world of clean, responsive, full-stack development as you learn what it takes to build a better web.</p>
-        <a href="apply-dev.html" class="button">Learn More</a>
+        <a href="apply-dev.html" class="button">Learn More & Apply</a>
```

Comment:
Do you mind if I encode this apostrophe?
```suggestion
        <a href="apply-dev.html" class="button">Learn More &amp; Apply</a>
```